addons/web_editor/static/src/js/snippets.editor.js

```diff
@@ -994,7 +994,9 @@ data.Editor = Class.extend({
         this.$target.after($clone);
         this.buildingBlock.call_for_all_snippets($clone, function (editor, $snippet) {
             for (var i in editor.styles) {
-                editor.styles[i].on_clone($snippet);
+                editor.styles[i].on_clone($snippet, {
+                    isCurrent: ($snippet.is($clone))
```

Comment:
We are writing JS guidelines here: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines
Here there is a missing trailing comma.
So:
```javascript
{
    isCurrent: $snippet.is($clone),
}
```